force-app/main/default/classes/DonationHistoryController.cls

```diff
@@ -77,7 +78,8 @@ public with sharing class DonationHistoryController {
     */
     @AuraEnabled(cacheable=true)
     public static DonationHistoryAppView getDonationHistory(Id contactId, Integer offset, String filter) {
-        DonationHistoryService service = new DonationHistoryService(contactId, offset, filter);
+        contactId = (contactId == null ? getContactIdByUserId() : contactId );
+		DonationHistoryService service = new DonationHistoryService(contactId, offset, filter);
```

Comment:
Fix indentation here, makes it seem like one long statement.

Reply:
Hi @lparrott. I do not understand the suggestion. To what exact statement are you referring to?

Reply:
Line 82 is indented with two tab characters instead of 8 spaces. We generally prefer to indent using 4 spaces per level of indentation.